src/views/v2/rsvp/messages/success.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Block: RSVP
+ * Messages Sucess
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/messages/success.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+$step = sanitize_text_field( tribe_get_request_var( 'step', '' ) );
+
+if ( 'success' !== $step ) {
+	return;
+}
+?>
+<div class="tribe-tickets__rsvp-message tribe-tickets__rsvp-message--success tribe-common-b3">
+	<em class="tribe-common-svgicon tribe-tickets__rsvp-message--success-icon"></em>
```

Comment:
use the svg variables directly instead, we plan to remove the `svgicon` classes and use svg templates for TEC.